rotkehlchen/tests/api/test_historical_balances.py

```diff
@@ -981,3 +982,45 @@ def test_historical_price_cache_only_special_assets(
     result = assert_proper_sync_response_with_result(response)
     assert str(START_TS) in result['prices']
     assert result['prices'][str(START_TS)] == '1400'
+
+
+@pytest.mark.parametrize('should_mock_price_queries', [False])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+@pytest.mark.parametrize('ethereum_accounts', [[]])
+def test_manual_prices_override_external_data(
+        rotkehlchen_api_server: 'APIServer',
+        gnosis_inquirer: 'GnosisInquirer',
+        globaldb: 'GlobalDBHandler',
```

Comment:
```suggestion
def test_manual_prices_override_external_data(
        rotkehlchen_api_server: 'APIServer',
        globaldb: 'GlobalDBHandler',
```
gnosis inquirer is unused now